README.md

```diff
@@ -466,6 +466,103 @@ Simbody is installed into the `usr/` directory.  The directory
     * `examples/src` source code for the examples.
     * `examples/bin` symbolic link to executable examples.
 
+Windows using MinGW
+-------------------
+
+Warning: The [MinGW](http://sourceforge.net/projects/mingw-w64/)
+generation and build is experimental!
+
+This build is still experimental, because of :
+
+* the various MinGW versions available (Thread model, exception mechanism)
+* the compiled libraries Simbody depends on (Blas and Lapack).
+
+If you do not want to go into details, you need a MinGW version with :
+
+* a Posix thread model and Dwarf exception mechanism on a 32 bit computer
+* a Posix thread model and SJLJ exception mechanism on a 64 bit computer
+
+Other versions are supported with additional configurations. Read the details below to understand everything.
+
+MinGW is available with two thread models :
+
+* Win32 thread model
+* Posix thread model
+
+One has to use the Posix thread model, since all thread functionalities (e.g. `std:mutex`) are not implemented.
+
+To ease building on Windows, Simbody provides compiled libraries for Blas and Lapack :
+
+* On Windows 32 Bits, these were compiled with a Dwarf exception mechanism,
+* On Windows 64 Bits, these were compiled with a SJLJ exception mechanism.
+
+If one chooses a MinGW compilation, we need to respect this exception mechanism.
+A program can not rely on both mechanisms.
+This means that if we want to use the compiled libraries, our MinGW installation should
+have the same exception mechanism.
+Otherwise, we need to provide our own Blas and Lapack libraries.
+
+To see which exception mechanism is used, user can look at dlls located in the `bin` directory of MinGW.
+The name of mechanism is present in the file `libgcc_XXXX.dll`, where `XXXX` can be `dw`, `seh` or `sljl`.
+For some MinGW version, this information is also available by looking at the result of `gcc --version`.
+
+CMake will check the version of your MinGW, and if the exception mechanism is different,
```

Comment:
Should mention that mingw bin directory must be on the PATH for things to work smoothly (assuming that's true).